docs/content/contributing/Logging.md

```diff
@@ -46,3 +46,28 @@ val logger = loggerFactory.create("MyCustomTag")
 `StubLoggerFactory` and `StubLogger` can be used in tests
 
 `StubLogger` will write to stdout only during test runs from IDE
+
+## Verbose mode
+
+Use gradle property `avito.logging.verbose` to override gradle logging level and send avito logs with gradle's error
```

Comment:
I didn't get the idea. 
Why do we need to make INFO or similar levels act like errors? What problem does it solve?
Maybe `avito.logging.verbose` confuses me

Reply:
Discussed. It's made to see all levels in stdout.